### PR TITLE
make tables accessible in mobile view.

### LINKED ISSUE #10704: [Docs] Tables in references is not accessible in mobile view.
### Current State

https://docs.meshery.io/reference/mesheryctl

screenshot_2024-04-03-12-31-30-58_720

### Desired State

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by Aseem22**
**Notes for Reviewers**

This PR fixes #10704

This PR make tables used at various pages of references scrollable.

https://github.com/meshery/meshery/assets/89309144/1c174d14-86b1-4e0c-a035-894ac177ec64

changes made at:
https://docs.meshery.io/reference/rest-apis
https://docs.meshery.io/reference/error-codes
https://docs.meshery.io/reference/mesheryctl

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 70e0b704f7c63932ccd5ae2e3e0fa972f526804a at: https://661c34b000cb3bb092a135bc--meshery-docs-preview.netlify.app

**Comment by Yashsharma1911** [comment]
Thankyou for this @Aseem22 checks are failing, also video format if not supported

File: docs/pages/reference/error-codes.md [hunk 1]
```diff
@@ -116,29 +121,28 @@ Note: The numeric portion of error codes are component-scoped. The numeric porti
...
-          {% capture heading %}
-            {{ component[1].component_name }} {{ component[1].component_type | camelcase }}
-          {% endcapture %}
-        {% elsif component[1].component_name == 'meshery-server' %}
-          {% capture heading %}
-            Meshery Server
-          {% endcapture %}
-        {% endif %}
-
+{% for files in site.data.errorref %}
```
Comment (leecalcote) [inline docs/pages/reference/error-codes.md:124]: @Aseem22 please revert this change in indentation.

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by vishalvivekm** [comment]
@Aseem22 Let's discuss this in Meshery dev call today. Add this as an agenda item to the meeting[ minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?pli=1#), if you would :)

**Comment by Aseem22** [comment]
> @Aseem22 Let's discuss this in Meshery dev call today. Add this as an agenda item to the meeting[ minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?pli=1#), if you would :)

Thanks @vishalvivekm. This issue has been discussed this monday on website meeting. I recieved some feedback which I am working on currently.

**Comment by vishalvivekm** [comment]
> > @Aseem22 Let's discuss this in Meshery dev call today. Add this as an agenda item to the meeting[ minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?pli=1#), if you would :)
> 
> Thanks @vishalvivekm. This issue has been discussed this monday on website meeting. I recieved some feedback which I am working on currently.

@Aseem22 
Do you have any updates on the issue ?
let's discuss this on today's website meeting at 5:30 PM IST. I am adding this as an [agenda item](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk). Please Join [https://meet.layer5.io/websites](https://www.google.com/url?q=http://meet.layer5.io/websites&sa=D&source=calendar&usd=2&usg=AOvVaw1CNpNwmYd_rTFcQa_7fBB5) if you are available during the meet.

**Comment by leecalcote** [comment]
PR abandoned.

**Pull request closed without merging**